src/util/moveItemInArray.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Shifts element array to designated position in array. fromIndex
+ * is the chosen array element and toIndex is the array position for
+ * you want the element to go
```

Comment:
"Shifts element *in an* array to *a* designated position."

The rest of this description should be the `@param` descriptions

Reply:
done

Reply:
> fromIndex
> is the chosen array element and toIndex is the array position for
> you want the element to go

That can all be removed since you're describing the parameters here, no need to describe parameters in the description of the function